src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -185,23 +182,20 @@ class TransactionDetailsComponent extends React.Component<Props, State> {
   }
 
   openAdvancedDetails = async () => {
-    const { currencyInfo, route } = this.props
+    const { wallet, route } = this.props
     const { edgeTransaction } = route.params
 
     Airship.show(bridge => (
-      <AdvancedDetailsModal
-        bridge={bridge}
-        transaction={edgeTransaction}
-        url={currencyInfo ? sprintf(currencyInfo.transactionExplorer, edgeTransaction.txid) : undefined}
-      />
+      <AdvancedDetailsModal bridge={bridge} transaction={edgeTransaction} url={sprintf(wallet.currencyInfo.transactionExplorer, edgeTransaction.txid)} />
     ))
   }
 
   renderExchangeData = (symbolString: string) => {
-    const { destinationDenomination, destinationWallet, guiWallet, walletDefaultDenomProps, theme, route } = this.props
+    const { destinationDenomination, destinationWallet, wallet, walletDefaultDenomProps, theme, route } = this.props
     const { edgeTransaction } = route.params
     const { swapData, spendTargets } = edgeTransaction
     const styles = getStyles(theme)
+    const walletName = wallet.name ?? sprintf(s.strings.my_crypto_wallet_name, wallet.currencyInfo.displayName)
```

Comment:
Use `getWalletName` for consistency across the app

Reply:
Oh, good catch!